UPGRADE.md

```diff
@@ -1,5 +1,10 @@
 # Upgrade to 3.0
 
+## BC BREAK: Remove YAML mapping drivers.
+
+If your code relies on `YamlDriver` or `SimpleYamlDriver`, you **MUST** migrate to
```

Comment:
You could point to the `orm:convert-mapping` command here as well:

```
php doctrine orm:convert-mapping xml /path/to/mapping-path-converted-to-xml
```

Reply:
Good idea.